### PR TITLE
Canceling cordoned app

### PR DESCRIPTION
**Pull request by tomahawk28**
Canceling on cordoned apps so we don't delete any configmap, secret and chart. 

## Checklist

- [x] Update changelog in CHANGELOG.md.

### CODE REVIEW

**Comment by tomahawk28** [comment]
Tested in `godsmack`

**Comment by tomahawk28** [comment]
```D 05/15 14:25:07 /apis/application.giantswarm.io/v1alpha1/namespaces/giantswarm/apps/app-operator-unique configmap.GetCurrentState app `app-operator-unique` is cordoned | app-operator/service/controller/app/resource/configmap/current.go:47 | controller=app-operator | event=update | loop=2768 | version=160322953
D 05/15 14:25:07 /apis/application.giantswarm.io/v1alpha1/namespaces/giantswarm/apps/app-operator-unique configmap.GetCurrentState canceling resource | app-operator/service/controller/app/resource/configmap/current.go:48 | controller=app-operator | event=update | loop=2768 | version=160322953
D 05/15 14:25:07 /apis/application.giantswarm.io/v1alpha1/namespaces/giantswarm/apps/app-operator-unique secret.GetCurrentState app `app-operator-unique` is cordoned | app-operator/service/controller/app/resource/secret/current.go:45 | controller=app-operator | event=update | loop=2768 | version=160322953
D 05/15 14:25:07 /apis/application.giantswarm.io/v1alpha1/namespaces/giantswarm/apps/app-operator-unique secret.GetCurrentState canceling resource | app-operator/service/controller/app/resource/secret/current.go:46 | controller=app-operator | event=update | loop=2768 | version=160322953
D 05/15 14:25:07 /apis/application.giantswarm.io/v1alpha1/namespaces/giantswarm/apps/app-operator-unique chart.GetCurrentState app `app-operator-unique` is cordoned | app-operator/service/controller/app/resource/chart/current.go:45 | controller=app-operator | event=update | loop=2768 | version=160322953
D 05/15 14:25:07 /apis/application.giantswarm.io/v1alpha1/namespaces/giantswarm/apps/app-operator-unique chart.GetCurrentState canceling resource | app-operator/service/controller/app/resource/chart/current.go:46 | controller=app-operator | event=update | loop=2768 | version=160322953
D 05/15 14:25:07 /apis/application.giantswarm.io/v1alpha1/namespaces/giantswarm/apps/app-operator-unique status app `app-operator-unique` is cordoned | app-operator/service/controller/app/resource/status/create.go:41 | controller=app-operator | event=update | loop=2768 | version=160322953
D 05/15 14:25:07 /apis/application.giantswarm.io/v1alpha1/namespaces/giantswarm/apps/app-operator-unique status canceling resource | app-operator/service/controller/app/resource/status/create.go:42 | controller=app-operator | event=update | loop=2768 | version=160322953```

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,6 +9,8 @@ and this project's packages adheres to [Semantic Versioning](http://semver.org/s
 
 ### Changed
 
+- Cancelling resources on cordoned app CRs.
```
Comment (rossf7) [inline CHANGELOG.md:12]:
```suggestion
- Cancel resources when app CRs are cordoned.
```

**Comment by tomahawk28** [comment]
@rossf7 Yes, I think it's best to cordon chart CRs for app-operator, chart-operator for precaution.

**Comment by rossf7** [comment]
@tomahawk28 Thanks. Plan is updated to also cordon the chart CRs.

**Pull request merged**